statechart/models/interpreter.py

```diff
@@ -7,15 +7,12 @@
 from sismic.interpreter import Interpreter as SismicInterpreter
 from sismic.model import Event
 
-from odoo.exceptions import except_orm
-
 
 def _root_cause(e):
-    if isinstance(e, except_orm):
-        return e
-    if not hasattr(e, "__cause__") or not e.__cause__:
-        return e
-    return _root_cause(e.__cause__)
+    cause_exc = getattr(e, "__cause__", False)
```

Comment:
```suggestion
    cause_exc = getattr(e, "__cause__", None)
```

A little detail, False for None is an odooism :)